Add ziti-ci tool and shared travis files

plorenz opened the pull request.

andrewpmartinez commented:
Part of me feels like we should make this repo private, regenerate the private keys and re-encrypt them.

I am not a fan of exposing private key files, even encrypted ones, that can be moved out of our control and brute forced.

plorenz commented:
> Part of me feels like we should make this repo private, regenerate the private keys and re-encrypt them.
> 
> I am not a fan of exposing private key files, even encrypted ones, that can be moved out of our control and brute forced.

We can skip the encrypted file and put the keys directly into travis, where only authorized can updated them and no one can view them after they've been set. I've updated the PR to follow this model and updated the other projects to follow that pattern. I also create a new key for ziti-ci and will remove the old one when the PRs have been pushed through.

I'm not sure what the benefit of making this repo private is... can you expand on that?
